projects/france/docker-compose.yml

```diff
@@ -74,9 +74,10 @@ services:
       - "./pelias.json:/code/pelias.json"
       - "${DATA_DIR}:/data"
   elasticsearch:
-    image: pelias/elasticsearch
+    image: pelias/elasticsearch:5.6.12
     container_name: pelias_elasticsearch
     restart: always
+    environment: [ "discovery.type=single-node" ]
```

Comment:
is it possible for us to set this as a default in the Dockerfile itself? It would be ideal to not have to specify it everywhere including in the schema documentation.

Reply:
I can try, actually I wanted to mention this setting to you, apparently, bash doesn't support exporting vars with a `.` in them (try it!) so it'll have to be done in either `elasticsearch.yml` or via a flag for the elasticsearch binary.

Reply:
ref: https://github.com/kubernetes/kubernetes/issues/2707

Reply:
Yikes, that's fun. I'm sure we can figure out a solution :)

Reply:
oh yea it's easy, I'll publish and remove the references to it

Reply:
resolved via rebase